src/styles/blocks/opinion.scss

```diff
@@ -0,0 +1,17 @@
+.stars-container {
+  display: flex;
+  align-items: center;
+}
+
+.starts {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__product-raiting-stars-4 :nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
here must be at least standart css styles for all six star in a row, and ideally should be a cycle.